stacker/status.py

```diff
@@ -77,6 +77,7 @@ class StackDoesNotExist(SkippedStatus):
 
 
 PENDING = PendingStatus()
+WAITING = PendingStatus(reason="waiting")
```

Comment:
It might be worth something like `waiting on previous update` for the reason, just to be more explicit.